src/yang/drned-xmnr.yang

```diff
@@ -272,12 +272,12 @@ module drned-xmnr {
               type uint16;
               default 120;
             }
-            leaf import-timeout {
-              tailf:info
-                "How long should NSO wait for the import command to complete.";
-              type uint16;
-              default 120;
-            }
+            //leaf import-timeout {
```

Comment:
I don't think we should keep it here, if we don't need it any longer.

Reply:
No, you're right. I meant to remove it before I pushed...